bcgov/ppr
# Mhr Filing Submission

*Issue #:* /bcgov/entity#13215

*Description of changes:*
- added a interface that matches the required properties in the API
- added a composable for constructing the request payload (we do not want to include `''`, `null`, or `undefined` values)
- some validation flags are set for testing purpose
- some properties are set for testing purpose
---
### Testing
- open up the console when clicking on `submit` to see the result
- use 83000007 or above for document id when testing, I increment 1 for every successful submission to avoid conflicts.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).
